### DISCUSSION TITLE
Adding custom request headers during detection is invalid for httpx

### DISCUSSION THREAD

**Discussion by sshenlian** [projectdiscovery/httpx#1819] [Q&A]
Adding custom request headers during detection is invalid for httpx，The order of changing parameters multiple times is also the same

<img width="1322" alt="image" src="https://github.com/projectdiscovery/httpx/assets/50750753/dad9bc94-e9e1-47d0-a209-e20e01024733">

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
